app/src/main/java/com/hedvig/app/feature/embark/util/MaskTypeUtil.kt

```diff
@@ -0,0 +1,25 @@
+package com.hedvig.app.feature.embark.util
+
+import java.time.LocalDate
+import java.time.Period
+import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeFormatterBuilder
+import java.time.temporal.ChronoField
+
+object MaskTypeUtil {
+
+    fun getYearsBetween(text: String, currentDate: LocalDate, pattern: String): String {
+        val dateTimeFormatter = createDateFormatterFromPattern(pattern)
+        val birthDate = LocalDate.parse(text, dateTimeFormatter)
+        return Period.between(birthDate, currentDate).years.toString()
+    }
+
+    private fun createDateFormatterFromPattern(pattern: String): DateTimeFormatter {
+        return DateTimeFormatterBuilder()
+            .appendValueReduced(
+                ChronoField.YEAR_OF_ERA, 2, 4, LocalDate.of(1940, 1, 1)
```

Comment:
If someone is born 1939 how would this behave?
If it acts weird, maybe we can bump this to 1925 at least, so that it handles all people born up to 2024? Or am I misunderstanding how this functions?